operator/v1/types_etcd.go

```diff
@@ -43,6 +43,16 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// BackendQuotaGiB sets the etcd backend storage size limit in gibibytes. Defaults to 8GiB.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=8
+	// +kubebuilder:validation:Maximum=32
```

Comment:
Why did the minimum disappear?

Reply:
@JoelSpeed  i used the `default` to replace `Minimum`

Reply:
No, I think you need a minimum still, otherwise I could create the object with a lower value